CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## TBC (TBC)
+
+### Enhancements
+
+* Files found with `Dir.glob` are now sorted alphabetically so that this action is deterministic accross OS's
+* Adds -m/--allow-missing-dwarf option to throw warnings instead of errors when a dSYM with missing DWARF data is found
+* Adds -e/--ignore-empty-dsym option to throw warnings instead of errors when a *.dSYM file is found rather than the expected *.dSYM directory"
```

Comment:
Can we cite some web references or give examples of when users may need these? Or put it in the PR description? Just want to make sure we keep the rationale.